k8s/distributed-chroma/templates/compaction-service.yaml

```diff
@@ -70,6 +70,15 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+          {{ if .Values.compactionService.resources }}
+          resources:
+            limits:
+              cpu: {{ .Values.compactionService.resources.limits.cpu }}
```

Comment:
What happens if the user sets part of resources but not all four?  Is the conditional doing more than being a partial measure against a broken manifest.

Reply:
Yes. I did note this but was following the existing pattern. The resulting yaml would be empty and k8s would error on application. 

I can cut a task to clean this up across the board.